Sources/TensorFlow/Loss.swift

```diff
@@ -86,6 +86,26 @@ public func categoricalHingeLoss<Scalar: TensorFlowFloatingPoint>(
     return max(Tensor(0), negative - positive + Tensor(1))
 }
 
+// Helper function for Logcosh
```

Comment:
Always use the full identifier to refer to a function, and end comments with a period.
```suggestion
// Helper function for `losCoshLoss(predicted:expected:)`.
```